src/sequence/pos_datadeps.jl

```diff
@@ -0,0 +1,31 @@
+function pos_datadep_register()
+    register(DataDep("POS Model Weights",
+        """
+        The weights for POS Sequence Labelling Model.
+        """,
+        "https://github.com/Ayushk4/POS.jl/releases/download/v0.0/weights.tar.xz",
+        "b02e891ea913be6834ff67d6ecf2ddae6754d55509bb3d9c078dbfc7eed27988";
+        post_fetch_method = function(fn)
+            unpack(fn)
+            dir = "weights"
```

Comment:
where will this go? Hopefully this won't clash with the NER weights?

Reply:
It won't clash with NER weights or dicts, since I named the datadeps for this block in `POS Model Weights` and `POS Model Dicts`.

As for NER, your model weights reside at path `~/.julia/datadeps/NER Model Weights/`, `~/.julia/datadeps/NER Model Dicts/`. While for POS it will be in `~/.julia/datadeps/POS Model Weights/` and `~/.julia/datadeps/POS Model Dicts/`. (For linux (ubuntu) specifically.) 
I am not aware of the paths in Windows, but DataDeps.jl should be managing this as well.

Reply:
Can I mark this resolved?